shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -200,37 +226,10 @@ static void RunBundleAndSnapshotFromLibrary(JNIEnv* env,
       fml::jni::JavaStringToString(env, jBundlePath))  // apk asset dir
   );
 
-  std::unique_ptr<IsolateConfiguration> isolate_configuration;
```

Comment:
this has to happen in 2 places now, so I moved it into the android shell holder as explained above.